d2plugin/plugin_features.go

```diff
@@ -50,5 +53,22 @@ func FeatureSupportCheck(info *PluginInfo, g *d2graph.Graph) error {
 			}
 		}
 	}
+	if _, ok := featureMap[DESCENDANT_EDGES]; !ok {
+		for _, e := range g.Edges {
+			// descendant edges are ok in sequence diagrams
```

Comment:
are they ok if one object isn't in a sequence diagram but the other is?

Reply:
I thought we didn't support edges from outside a sequence diagram to inside one? Is there a test that does this?

Reply:
Ah, if we don't then all is good.